tasks/apt.yml

```diff
@@ -1,6 +1,8 @@
 ---
-- name: remove packages
-  apt: name='{{item}}' state=removed
+- name: remove deprecated or insecure packages | package-01 - package-09
+  apt:
+    name: '{{item}}'
+    state: 'absent'
   with_items:
-    - "{{os_security_packages_list}}"
-  when: os_security_packages_clean
+    - '{{os_security_packages_list}}'
+  when: 'os_security_packages_clean'
```

Comment:
`os_security_packages_clean` is supposed to be a boolean. You can ensure this with the `bool` filter :)